homeassistant/components/tile/config_flow.py

```diff
@@ -56,3 +57,43 @@ async def async_step_user(
             return await self._show_form({"base": "invalid_auth"})
 
         return self.async_create_entry(title=user_input[CONF_USERNAME], data=user_input)
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: config_entries.ConfigEntry,
+    ) -> TileOptionsFlowHandler:
+        """Get the options flow for this handler."""
+        return TileOptionsFlowHandler(config_entry)
+
+
+class TileOptionsFlowHandler(config_entries.OptionsFlow):
+    """Handle Tile options."""
+
+    def __init__(self, config_entry: config_entries.ConfigEntry) -> None:
+        """Initialize Tile options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Manage Tile options."""
+        if user_input is not None:
+            self.hass.async_create_task(
+                self.hass.config_entries.async_reload(self.config_entry.entry_id)
+            )
+            return self.async_create_entry(title="", data=user_input)
+
+        return self.async_show_form(
+            step_id="init",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(
+                        CONF_SHOW_INACTIVE,
+                        default=self.config_entry.options.get(
+                            CONF_SHOW_INACTIVE, DEFAULT_SHOW_INACTIVE
+                        ),
+                    ): bool
```

Comment:
But wait, the tile itself is considered a device? We have a construct in Home Assistant for users being able to disable devices they don't use. I feel like this is adding the same construct in a new way?

Reply:
Users can use the Tile app to disable a Tile; that's what this PR is intended to "solve" (hiding Tiles that have been disabled in their app). The API doesn't have a way to disable the Tile, so even though these use cases are parallel, I'm thinking they're still separate?

Reply:
I was not talking about the Tile API or device, I was talking about a device in Home Assistant. A DeviceEntry can be disabled.

![image](https://user-images.githubusercontent.com/195327/141694183-d86ffbd3-0ff5-4d12-9c04-3e7dbfc0e254.png)

Reply:
Right. And that will accomplish the same thing. If this PR is merged in, though, disabled Tiles will be ignored automatically.

Reply:
Was literally about to say the same thing as @bachya. There's no concept from Tile for deleting devices you no longer have, such as the first tile trackers that didn't have easy battery replacement. Instead they are "replaced" and just marked as hidden. So it unnecessarily clutters Home Assistant by default because they still exist in the API.

Reply:
> Right. And that will accomplish the same thing.

yes, so that is my point against this PR, it introduces another construct. Instead, we should use what Home Assistant already has and offers.

We don't need an option flow for that.

Reply:
I'm open. 🤷🏻 @natekspencer Do you see anything that the built-in functionality _wouldn't_ handle?

Reply:
I'm open as well. I guess we could rewrite it to have the devices disabled by default?

Reply:
That's a good idea: look at the API results and for each Tile we see that's disabled, disable the corresponding HASS device. We would need to add device info for Tiles, though – doesn't exist today.